test/functional/custom-decorators.spec.ts

```diff
@@ -87,6 +88,16 @@ describe("custom decorators", function() {
                 errors[0].constraints.should.be.eql({ isLongerThan: "firstName must be longer then lastName. Given value: Li" });
             });
         });
+
+        it("should include context", function() {
```

Comment:
Could you please move this test to `test/functional/validation-options.spec.ts` where all tests with context feature are?

Reply:
👍 sounds good! Done in https://github.com/typestack/class-validator/pull/296/commits/e68b65eb37c11af4fa428aafde5182c00b787908